components/image/slider/src/index.js

```diff
@@ -39,6 +48,8 @@ const getSlides = (currentSlide, images = [], linkFactory) => {
         key={key}
         src={src}
         title={title}
+        height={height}
+        width={width}
```

Comment:
🍆  What happens if the height and/or width are `undefined`?

Reply:
this attribute won't be printed